bookwyrm/templates/guided_tour/direct_messages.html

```diff
@@ -0,0 +1,81 @@
+{% load i18n %}
+
+<script>
+    const tour = new Shepherd.Tour({
+        exitOnEsc: true,
+    });
+
+    tour.addSteps([
+        {
+            text: "{% trans 'You can send a Message to other Bookwyrm users. To send a message, use the <strong>@</strong> symbol with their username(s), followed by your message.<br><br>For example: <strong>@mouse Hi Mouse, I love reading books too!</strong>' %}",
```

Comment:
I think it's better to skip the example, since it won't make a lot of sense to people who don't know who I am, and if there's a poor unsuspecting user on another instance with that username, they might get some odd DMs

Reply:
Overall, I wonder if it would be better to do the post status box on the book page rather than DMs? DMs are less commonly, and the book-related statuses are more platform-specific

Reply:
Yep makes sense, will adjust that. It's useful to have made this step though because I found a bug lol.